client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -119,6 +120,23 @@ class ManageMenu extends PureComponent {
 				wpAdminLink: 'edit-comments.php',
 				showOnAllMySites: false,
 			},
+			{
+				name: 'plugins',
+				label: translate( 'Plugins' ),
+				capability: 'manage_options',
```

Comment:
@apeatling did we get the ✅ to move this from Configure subsection to Manage?

Reply:
Yup, the idea was move the `Plugins` item into the manage section.